index.html

```diff
@@ -1510,7 +1581,7 @@ <h4>Recommended Practice</h4>
 
     <!-- Protocol Binding -->
     <section id="protocol-binding">
-      <h3>Protocol Binding</h3>
+      <h3>HTTP 1.1 Protocol Binding</h3>
```

Comment:
At this point the version number is just added to avoid any ambiguities.
I'm not sure if HTTP/2 will require any other changes beyond updated references.